lib/prometheus/client/push.rb

```diff
@@ -36,6 +36,10 @@ def replace(registry)
         request('PUT', registry)
       end
 
+      def delete
+        request('DELETE')
```

Comment:
Given we're not going to send the registry, I guess we shouldn't pretend that in the Content-Type header. Wdyt about just calling `@http.send_request(...)` directly here?